src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -27,6 +27,56 @@ internal JsonElement(JsonDocument parent, int idx)
             _idx = idx;
         }
 
+        /// <summary>
+        ///   Parses one JSON value (including objects or arrays) from the provided reader.
+        ///   For performance, the JsonDocument Parse() methods should be used instead when the
```

Comment:
This "For performance" sentence is the only diff from the equivalent JsonDocument.Parse method.